packages/clerk-js/bundlewatch.config.json

```diff
@@ -4,7 +4,7 @@
     { "path": "./dist/clerk.headless.js", "maxSize": "44kB" },
     { "path": "./dist/ui-common*.js", "maxSize": "87KB" },
     { "path": "./dist/vendors*.js", "maxSize": "70KB" },
-    { "path": "./dist/coinbase*.js", "maxSize": "58KB" },
+    { "path": "./dist/coinbase*.js", "maxSize": "59KB" },
```

Comment:
Isn't it weird that the changes in this PR affect the `coinbase` chunk (?) cc @nikosdouvlis

Reply:
@anagstef  I did some clean ups earlier and the max size is  < 58KB. I will update the bundlewatch config